### ISSUE TITLE
Request failed with status 400

### ISSUE THREAD

**Issue by stpaultim** [zulip/hubot-zulip#12]
I'm trying to debug an error I'm getting on Heroku installation of Hubot-Zulip. I would appreciate any help.

```
2020-01-05T22:03:08.000000+00:00 app[api]: Build succeeded
2020-01-05T22:03:09.332709+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2020-01-05T22:03:09.396322+00:00 heroku[web.1]: Process exited with status 0
2020-01-05T22:03:10.957496+00:00 heroku[web.1]: Starting process with command `bin/hubot -a zulip`
2020-01-05T22:03:15.139887+00:00 app[web.1]: audited 213 packages in 1.641s
2020-01-05T22:03:15.141877+00:00 app[web.1]: found 11 vulnerabilities (9 moderate, 2 high)
2020-01-05T22:03:15.141880+00:00 app[web.1]: run `npm audit fix` to fix them, or `npm audit` for details
2020-01-05T22:03:15.902399+00:00 heroku[web.1]: State changed from starting to up
2020-01-05T22:03:15.678722+00:00 app[web.1]: Zulip API: Error parsing JSON: <h1>Bad Request (400)</h1>
2020-01-05T22:03:15.678986+00:00 app[web.1]: Zulip API Error: Request failed with status 400: <h1>Bad Request (400)</h1>
2020-01-05T22:03:15.681443+00:00 app[web.1]: [Sun Jan 05 2020 22:03:15 GMT+0000 (Coordinated Universal Time)] ERROR Error: Request failed with status 400: <h1>Bad Request (400)</h1>
2020-01-05T22:03:15.681446+00:00 app[web.1]: at Request._callback (/app/node_modules/zulip/lib/zulip.js:64:19)
2020-01-05T22:03:15.681448+00:00 app[web.1]: at Request.self.callback (/app/node_modules/request/request.js:129:22)
2020-01-05T22:03:15.681451+00:00 app[web.1]: at Request.emit (events.js:209:13)
2020-01-05T22:03:15.681453+00:00 app[web.1]: at Request.<anonymous> (/app/node_modules/request/request.js:873:14)
2020-01-05T22:03:15.681455+00:00 app[web.1]: at Request.emit (events.js:214:15)
2020-01-05T22:03:15.681457+00:00 app[web.1]: at IncomingMessage.<anonymous> (/app/node_modules/request/request.js:824:12)
2020-01-05T22:03:15.681459+00:00 app[web.1]: at IncomingMessage.emit (events.js:214:15)
2020-01-05T22:03:15.681461+00:00 app[web.1]: at endReadableNT (_stream_readable.js:1178:12)
2020-01-05T22:03:15.681463+00:00 app[web.1]: at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by stpaultim** [comment]
I got it working. The documentation here says to set the following environmental variables:

```
heroku config:add HUBOT_ZULIP_BOT=hubot-bot@example.com
heroku config:add HUBOT_ZULIP_API_KEY=your_key
```

However, this documentation adds a third variable. I was missing the third variable. 
https://backdrop.zulipchat.com/integrations/doc/hubot

```
export HUBOT_ZULIP_SITE="https://site.zulipchat.com/api"
export HUBOT_ZULIP_BOT="hubot-bot@example.com"
export HUBOT_ZULIP_API_KEY="your_key"
```

**Comment by stpaultim** [comment]
Here is a PR - https://github.com/zulip/hubot-zulip/pull/13

**Comment by timabbott** [comment]
Fixed in #13, thanks for the report and fix!
